tests/fixtures/celery.py

```diff
@@ -142,3 +157,199 @@ def celery_fixture(
             celery_session_worker,
             mock_session_maker,
         )
+
+
+class ApplyTaskFixture:
```

Comment:
This fixture is a more generic version of the previous `Opds2ImportFixture`. I pulled out the parts of that fixture that seemed like they would be generally useful when testing code that queues up `bibliographic_apply` and `circulation_apply` tasks. This fixture gets used when testing all of our OPDS import tasks.